starknet_devnet/blueprints/base.py

```diff
@@ -221,3 +221,37 @@ async def fork_status():
     if config.fork_network:
         return jsonify({"url": config.fork_network.url, "block": config.fork_block})
     return jsonify({})
+
+
+@base.route("/abort_blocks", methods=["POST"])
+async def abort_blocks():
+    """Abort blocks and transactions from given block hash to last block."""
```

Comment:
So far we haven't had the practice to put this much logic into a blueprint (blueprints/base.py in this case), usually starknet_wrapper.py, blocks.py and transactions.py were the ones holding the logic. Although this logic does seem correct.

Reply:
I'm changing that

Reply:
Fixed 82d227e5090031ced7f3aff2eabffcf5803b8173